### ISSUE TITLE
Plugin does not recognise Interface range for rstp and lldp

### ISSUE THREAD

**Issue by niklash-dev** [codeout/vscode-junos#3]
The Plugin does not recognise Interface ranges for rstp and lldp. As an example please past the following lines in VS Code:

`set protocols rstp interface uplink mode point-to-point`
`set protocols rstp interface edge-group edge`
`set protocols layer2-control bpdu-block interface edge-group`
`set protocols lldp interface all`
`set protocols lldp-med interface all`

You should see "uplink", "edge-group" and "all" highlighted in red.

The config I used runs with no errors on Junos devices.

Thank you for your work!

**Comment by codeout** [comment]
Thanks for your feedback.

vscode-junos doesn't support interface-range today, but it isn't difficult to implement.

```
# definition
set interfaces interface-range uplink member xe-*/0/0

# reference
set protocols rstp interface uplink mode point-to-point
```

**Comment by Administratoor** [comment]
I am also interested in the "all" interface-range :+1: 
Will there be an code update in the future?

**Comment by codeout** [comment]
hi @macbinjo, @Administratoor
I'm done implementing `interface-range` and finally published v0.2.3 with the feature.

https://user-images.githubusercontent.com/921696/140652252-6a695af7-5e07-49e5-a0c3-424bd4cc51d0.mov

Please reopen this issue when you see any further problems.
Thanks again for your feature request!
